2996_reflection/reflection.md

```diff
@@ -7756,14 +7760,26 @@ constexpr info r1 = substitute(^^fn1, {^^int});
 
 template <typename T>
 auto fn2() {
-  static_assert(^^T != ^^int); // static assertion failed during instantiation of fn<int>
+  static_assert(^^T != ^^int);
+    // static assertion failed during instantiation of fn<int>
   return 0;
 }
 
 constexpr bool r2 = can_substitute(^^fn2, {^^int});
   // error: instantiation of body of fn<int> is needed to deduce return type
 ```
 :::
+
+[#]{.pnum}
+
+::: example
+```cpp
+consteval info to_integral_constant(unsigned i) {
+  return substitute(^^integral_constant, {^^unsigned, reflect_value(i)});
+    // OK, represents integral_constant<unsigned, i>
```

Comment:
I think better to add a variable that's initialized with `to_integral_constant(2)` and say that it represents the type `integral_constant<unsigned, 2>`

Reply:
Great idea, done.